src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -879,9 +1027,19 @@ internal void SearchAndUpdateResults(string query)
             // if last sync isn't populated, we can't search
             if (LastSync == null) return;
 
+            IEnumerable<PackageManagerSearchElementViewModel> results;
             this.SearchText = query;
 
-            var results = Search(query);
+            // If the search query is empty, just call regular search API
+            // If the search query is not empty, then call Lucene search API
+            if (string.IsNullOrEmpty(query))
+            {
+                results = Search(query);
```

Comment:
any reasons for calling the old API? I think we do plan to obsolete that later

Reply:
@reddyashish Any update on this one?

Reply:
yeah i am removing the calls to this search API.

Reply:
Removed the old search(). Added a new method to get all packages(except deprecated) when the search query is empty.